src/global-table.ts

```diff
@@ -103,9 +117,14 @@ export class GlobalTable extends GlobalTableBase {
         attributeName: props.partitionKey.name,
         keyType: 'HASH',
       }],
-      replicas: [{
-        region: Stack.of(scope).region,
-      }],
+      streamSpecification: {
+        streamViewType: 'KEYS_ONLY',
+      },
```

Comment:
Add a section to the readme for this feature, and explain that stream specification is always `KEYS_ONLY` if a replica is supplied by the user. 

That will need to change in the future, customers will want to be able to choose the stream type.